spec/javascripts/packages/document-capture/hooks/use-i18n-spec.jsx

```diff
@@ -46,6 +59,28 @@ describe('document-capture/hooks/use-i18n', () => {
 
       expect(container.childNodes).to.have.lengthOf(2);
     });
+
+    it('allows (but discards) attributes in the input string', () => {
+      const formatted = formatHTML(
+        '<strong data-first-before>Hello</strong> <strong data-second-before>world</strong>',
+        {
+          strong: ({ children }) => <strong data-after>{children}</strong>,
+        },
+      );
+
+      const { container } = render(formatted);
+
+      expect(container.querySelectorAll('[data-after]')).to.have.lengthOf(2);
+      expect(container.querySelectorAll('[data-before]')).to.have.lengthOf(0);
```

Comment:
Neither of the elements have `data-before`, should this be this instead?

```suggestion
      expect(container.querySelectorAll('[data-first-before]')).to.have.lengthOf(0);
      expect(container.querySelectorAll('[data-second-before]')).to.have.lengthOf(0);
```

Reply:
Ah. I had changed the test to verify against potential issues with regex laziness, adding a second `<strong>` tag to the input string. I apparently forgot to fully update the assertions accordingly.

Taking a closer look, I don't think the `first-` and `second-` are really necessary for this, so I've removed them in 139f07816.